nilearn/plotting/js_plotting_utils.py

```diff
@@ -120,8 +120,8 @@ def get_iframe(self, width=None, height=None):
         if height is None:
             height = self.height
         escaped = escape(self.html, quote=True)
```

Comment:
The function `escape()` should be renamed to something self-documenting, such as `replace_html_unsafe_chars()`. Rename the variable `escape` with something self-documenting as well, `safe_html` for example.

Reply:
the function `escape` is from the standard library. reserved characters are "escaped", hence the name.

Reply:
The docstring for `escape()` states:
```
    Replace special characters "&", "<" and ">" to HTML-safe sequences.
    If the optional flag quote is true (the default), the quotation mark
    characters, both double quote (") and single quote (') characters are also
    translated.
```
It can be cleaned up thusly:
```
    Replaces characters "&", "<", ">" to HTML-safe (characters?)sequences.
    If quote=True (default), double quote (") and single quote (') characters are also
    replaced.
```

Reply:
Okay, Standard Library. Still, terrible name IMO. Removing my comment.